### PR TITLE
Enhance OS check issues templates

### PR DESCRIPTION
**Pull request by jotacarma90**
|Related issue|
|---|
|https://github.com/wazuh/wazuh/issues/23903|

# Description
Hi team, this PR is to improve the current OS check related templates, to be able to open all the necessary issues using those templates. One for the epic issue, and another for all the children, uncommenting those blocks that are necessary for each issue, taking into account the components to test.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/planned--specific-support-new-oss.md.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- [ ] **Major/New family**: Add support in the allocator. This is the first step for everything else, regardless of the team as we need to be able to allocate OS VMs to generate a testing environment.
+- [ ] **Minor**: Update the allocator images.
+- [ ] **Central components**: Review/test the installation assistant using the new OS.
+- [ ] Update AMI, OVA, or Docker images if needed.
+-->
+
+<!-- Uncomment for CPPSERVER issue
+**CppServer**
+- [ ] **Central components**: Add support for the new OS to the GitHub Actions package builder.
+- [ ] **Central components**: Smoke test that the package works, including installation and upgrade.
```
Comment (MarcelKemp) [inline .github/ISSUE_TEMPLATE/planned--specific-support-new-oss.md.md:38]: We always rely on our internal documentation to update the template as well:
```suggestion
- [ ] **Central components**: Smoke test that the package works, including installation and upgrade.
- [ ] **Major/New family**: Make sure that VD works properly according to the OS tier.
```

File: .github/ISSUE_TEMPLATE/planned--specific-support-new-oss.md.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+<!-- Uncomment for QA issue
+**QA**
+- [ ] **Major/New family**: Add new tests according to the OS's tier.
+- [ ] **Major/New family**: Add proper documentation.
+-->
+
+<!-- Uncomment for DEVOPS issue
+**DevOps**
+- [ ] **Major/New family**: Add support in the allocator. This is the first step for everything else, regardless of the team as we need to be able to allocate OS VMs to generate a testing environment.
```
Comment (MarcelKemp) [inline .github/ISSUE_TEMPLATE/planned--specific-support-new-oss.md.md:29]:
```suggestion
- [ ] **Major/New family**: Add support in the allocator. This is the first step for everything else, regardless of the team as we need to be able to allocate OS VMs to generate a testing environment.
- [ ] **Major/New family**:  Adapt Puppet.
- [ ] **Major/New family**: Adapt Ansible.
```

**Review by MarcelKemp — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/planned--specific-support-new-oss.md.md (hunk 1 above)
Comment (jotacarma90) [reply .github/ISSUE_TEMPLATE/planned--specific-support-new-oss.md.md:38]: done

File: .github/ISSUE_TEMPLATE/planned--specific-support-new-oss.md.md (hunk 2 above)
Comment (jotacarma90) [reply .github/ISSUE_TEMPLATE/planned--specific-support-new-oss.md.md:29]: done

**Review by MarcelKemp — APPROVED** [review]

**Review by lchico — APPROVED** [review]
LGTM!

**Review by mjcr99 — APPROVED** [review]
LGTM

**Review by MarcelKemp — CHANGES_REQUESTED** [review]
The document has been updated, so we need to update the template again.

**Review by lchico — APPROVED** [review]

**Pull request merged**